### PR TITLE
remove pytest pinning on appveyor

### PR DESCRIPTION
**Pull request by kain88-de**
already done for travis in #257

### CODE REVIEW

File: appveyor/install-miniconda.ps1 [hunk 1]
```diff
@@ -168,9 +168,6 @@ python --version
 checkLastExitCode
 
 # CORE DEPENDENCIES
-# Remove pytest version limitation once
-# https://github.com/astropy/astropy/pull/6419 is solved
-Copy-Item ci-helpers\appveyor\pinned ${env:PYTHON}\envs\test\conda-meta\pinned
```
Comment (bsipocz) [inline appveyor/install-miniconda.ps1]: I would keep this as a placeholder (but remote the 2 comment lines above), and just remove the pytest line from that file

**Comment by bsipocz** [comment]
Thanks @kain88-de! Due to master being a protected branch, this will need to be rebased to rerun the tests. Also please address the minor comment I left.

File: appveyor/install-miniconda.ps1 (hunk 1 above)
Comment (kain88-de) [reply appveyor/install-miniconda.ps1]: I added a comment that pins should be added to this file. The pytest bin is now also removed from the file. Not sure how well git treats empty files though. I never tried that.

**Comment by bsipocz** [comment]
Thanks @kain88-de!

**Pull request merged**